tests/h/session_test.py

```diff
@@ -249,18 +240,18 @@ def test_format_returns_empty_dict_when_user_missing(self):
         assert session.user_info(None) == {}
 
 
-class FakeAuthorityGroupService(object):
+class FakeListGroupsService(object):
```

Comment:
D'oh. I thought I killed this. I'll fix in a subsequent PR (as I have a few other test cleanups to tackle).